app/src/ui/octicons/sync-clockwise.ts

```diff
@@ -1,17 +1,17 @@
-import { OcticonSymbolType } from './octicons.generated'
+import { OcticonSymbolVariant } from '.'
 
 /**
  * An horizontally flipped (i.e `scaleX(-1)`) but otherwise unmodified version of
  * the `sync` octicon.
  */
-export const syncClockwise: OcticonSymbolType = {
+export const syncClockwise: OcticonSymbolVariant = {
```

Comment:
Curious, could we make the two custom octicons (this one and `diffCheck`) be `OcticonSymbolVariants` type and simplify our `octicon.tsx` code since we wouldn't need to check for single versus a map. Just always expect as map and these one's just happen to have 1 size in the map.

Reply:
I see we have a few other scattered through the files too. :P I guess we do not have a very consistent pattern on where we store custom octicons. :P

Reply:
@tidy-dev Yeah I was going to say something similar. I think being able to define a single custom variant and pass that in with no need to nest the custom icon in another level of a hash is good compromise. I did go back and forth a couple of times.